jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_sorting_view.py

```diff
@@ -93,8 +93,8 @@ def _handle_message(self, widget, msg, buffers):
             elif msg['type'] == 'addSubfeedWatch':
                 self._worker_session.add_subfeed_watch(
                     watch_name=msg['watchName'],
-                    feed_id=msg['feedId'],
-                    subfeed_hash=msg['subfeedHash']
+                    feed_uri=msg['feedUri'],
```

Comment:
When communicating between server/client we are going to use feedUri and subfeedName rather than feedId and subfeedHash. That's because the latter can be derived from the former, but not the other way around.